tests/lib/modem_trace/prj.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_UNITY=y
+CONFIG_NO_OPTIMIZATIONS=y
```

Comment:
why are optimization not required? is this relevant?

Reply:
When debugging, it is nice to have the optimizations off. No point in optimizing unit tests anyway.

Reply:
> When debugging.

But users can build the test without optimization locally when needed.
Thus it doesn't need to be part of `prj.conf`.

Reply:
I still dont see why this needs to be removed because it does no harm and only helps when writing and debugging test failures. But I give in and will remove this from here since there are two people opposing this idea.

Reply:
I suppose the idea is to have a consistent set of configs across samples/tests (as much as it's possible), as opposed to every developer creating project files based on what they use instead. I think this is particularly the case for settings that are applicable to all projects, for example the optimization level.

Reply:
This is now removed.